upgrade/run_upgrade_test.sh

```diff
@@ -49,9 +49,6 @@ export TARGET_LINUX_TAR_SUFFIX=${TARGET_LINUX_TAR_SUFFIX:-"linux-amd64.tar.gz"}
 # to downgrade to.
 export TEST_SCENARIO=${TEST_SCENARIO:-"upgrade-downgrade"}
 
-# obtain the actual source tag
-SOURCE_TAG=$(curl -s https://storage.googleapis.com/istio-build/dev/"$SOURCE_TAG")
-
```

Comment:
Hi @linsun, we process this at https://github.com/istio/tools/blob/86ed08918b5682666e83c8b2e69b798cdf371466/upgrade/run_upgrade_test.sh#L65 so i just remove this line here. Please take a look, thanks!

Reply:
and same for release-1.7 branch? https://github.com/istio/tools/blob/release-1.7/upgrade/run_upgrade_test.sh#L53

Reply:
Yes, agreed.